src/engraving/libmscore/gradualtempochange.cpp

```diff
@@ -151,7 +151,6 @@ LineSegment* GradualTempoChange::createLineSegment(System* parent)
 SpannerSegment* GradualTempoChange::layoutSystem(System* system)
 {
     SpannerSegment* segment = TextLineBase::layoutSystem(system);
-    moveToSystemTopIfNeed(segment);
 
     return segment;
 }
```

Comment:
This basically removes the need for this whole override, doesn't it?

Reply:
Oh, excellent catch. It sure does!! I'll remove it in a bit (currently working on a different branch)